CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 ## 3.56.0 (Unreleased)
+FEATURES:
+* **New Resource:** `google_privateca_certificate` ([#2924](https://github.com/hashicorp/terraform-provider-google-beta/pull/2924))
+
+IMPROVEMENTS:
+* bigquery: added `status` field to `google_bigquery_job` ([#2926](https://github.com/hashicorp/terraform-provider-google-beta/pull/2926))
+* compute: added `disk.resource_policies` field to resource `google_compute_instance_template` ([#2929](https://github.com/hashicorp/terraform-provider-google-beta/pull/2929))
+* provider: added plan time validations for fields that expect base64 values. ([#2906](https://github.com/hashicorp/terraform-provider-google-beta/pull/2906))
```

Comment:
I think this should be `* all:` instead of provider

Reply:
Fixed. It looks like both are used - what makes this one all instead of provider?

Reply:
Hm, I thought we just used `all:`. No preference then if we have both